webapp/channels/src/components/admin_console/server_logs/logs.tsx

```diff
@@ -144,16 +147,27 @@ export default class Logs extends React.PureComponent<Props, State> {
                         <FormattedMessage {...messages.bannerDesc}/>
                     </div>
                 </div>
-                <button
-                    type='submit'
-                    className='btn btn-primary'
-                    onClick={this.reloadPlain}
-                >
-                    <FormattedMessage
-                        id='admin.logs.ReloadLogs'
-                        defaultMessage='Reload Logs'
-                    />
-                </button>
+                <div className='banner-buttons'>
+                    <button
+                        type='submit'
+                        className='btn btn-primary'
+                        onClick={this.reloadPlain}
+                    >
+                        <FormattedMessage
+                            id='admin.logs.ReloadLogs'
+                            defaultMessage='Reload Logs'
+                        />
+                    </button>
+                    <ExternalLink
```

Comment:
The final CI failure is because these `ExternalLink` components are missing a `location` prop. I think you can just set that to `'download_logs'` to make the type checker happy